### PR TITLE
add Airflow Provider blog post and images

### PR DESCRIPTION
**Pull request by merobi-hub**
This adds a blog post about the release of the OpenLineage Airflow Provider.

### CODE REVIEW

**Comment by merobi-hub** [comment]
screencapture-localhost-3000-blog-airflow-provider-2023-08-23-11_49_12

File: blog/airflow-provider/index.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Improvements coming with the provider are not limited to fixes, however. The OpenLineage Provider promises to enable some long-sought-after enhancements, including support for one of the most-used Airflow operators – more about which below. 
+
+### High-level Design
+
+The provider approach solves these maintenance and reliability issues by moving the extraction logic, along with unit tests, to each provider. Although a lot of up-front work has gone into creating the provider, full implementation of this solution has actually been distributed (and necessarily remains a work in progress). No longer self-contained, the integration is now part of the operator contract and belongs to every provider that supports OpenLineage. Relocating the extraction logic in this way makes the integration more robust by ensuring the stability of the lineage contract in each operator. Another benefit of the approach: adding lineage coverage to custom operators is now easier.
```
Comment (julienledem) [inline blog/airflow-provider/index.mdx:35]: this is a great summary

File: blog/airflow-provider/index.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+### Implementation
+
+The OpenLineage Provider has been implemented in Airflow by reimplementing the [`openlineage-airflow`](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow) package from the OpenLineage project in the `apache-airflow-providers-openlineage` provider in the base Airflow Docker image, where it can be easily enabled by configuration. Furthermore, lineage extraction logic that was included in [Extractors](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow/openlineage/airflow/extractors) in that package is now implemented in operators living in their provider package along with unit tests, eliminating the need for Extractors in most cases. For this purpose, a new optional API for Operators (`get_openlineage_facets_on_{start(), complete(ti), failure(ti)}`, documented [here](https://openlineage.io/docs/integrations/airflow/default-extractors)) can be used.
```
Comment (julienledem) [inline blog/airflow-provider/index.mdx:39]: " provider in the base Airflow Docker image" I think this waits for 2.7.1 or something.

**Review by julienledem — APPROVED** [review]
This is great!

**Pull request merged**